### ISSUE TITLE
Missing libgfortran dependency in numpy 1.10.4

### ISSUE THREAD

**Issue by lesteve** [conda/conda#2028]
To reproduce:

```bash

conda create -n tmp python numpy=1.10.4 scipy -y
. activate tmp
python -c 'import scipy.linalg'
```

Output:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/lesteve/miniconda3/envs/tmp/lib/python3.5/site-packages/scipy/linalg/__init__.py", line 174, in <module>
    from .misc import *
  File "/home/lesteve/miniconda3/envs/tmp/lib/python3.5/site-packages/scipy/linalg/misc.py", line 5, in <module>
    from .blas import get_blas_funcs
  File "/home/lesteve/miniconda3/envs/tmp/lib/python3.5/site-packages/scipy/linalg/blas.py", line 155, in <module>
    from scipy.linalg import _fblas
ImportError: libgfortran.so.1: cannot open shared object file: No such file or directory
```

The same snippet works fine with numpy=1.10.2.

Looking at the output of conda install numpy=1.10.2, it seems like numpy 1.10.2 depends on libgfortran but numpy 1.10.4 does not.

**Comment by cpaulik** [comment]
I have the same problem with Python 2.7 but strangely enough not with 3.4. See my [travis build](https://travis-ci.org/TUW-GEO/pytesmo/builds/107213723).

When installing `numpy=1.10.2` into an environment where `numpy=1.10.4` is installed `libgfortran` and `openblas` are installed.

**Comment by lesteve** [comment]
> I have the same problem with Python 2.7 but strangely enough not with 3.4. See my travis build.

The snippet above fails consistently on all versions of python (just replace python by python=3.4 or python=2.7 for example).

**Comment by dpryan79** [comment]
At least for me this works on OSX but not Linux. Quite annoying.

**Comment by jreback** [comment]
cc @ilanschnell @msarahan

**Comment by jakevdp** [comment]
This is breaking my CI tests as well :cry: https://travis-ci.org/jakevdp/wpca/jobs/107271307

**Comment by hgomersall** [comment]
Yes, breakage over my way too: https://travis-ci.org/pyFFTW/pyFFTW

Though 3.3 seems to be working fine.

**Comment by ilanschnell** [comment]
I'm working on fixing this right now.

**Comment by ilanschnell** [comment]
Should be fixed now.

**Comment by Eric89GXL** [comment]
I'll can a failing Travis build to check. Has it been pushed to `Miniconda-Latest-...`?

**Comment by Eric89GXL** [comment]
I'll restart a failing build, rather... our `.travis.yml` does `conda update --yes --quiet conda` if that's also expected to fix it.

**Comment by Eric89GXL** [comment]
Looks like it is indeed fixed. Thanks for the quick response!

**Comment by dpryan79** [comment]
@ilanschnell :+1: Don't suppose you can fix the default curl package too (https://github.com/ContinuumIO/anaconda-issues/issues/72)? You can just copy the build recipe from bioconda (https://github.com/bioconda/bioconda-recipes/tree/master/recipes/curl).

**Comment by jakevdp** [comment]
Works for me – Many thanks!

**Comment by patricksnape** [comment]
scipy 0.16.1 is broken in the same way! See [here](https://travis-ci.org/menpo/menpo/builds/107303109)

**Comment by ilanschnell** [comment]
scipy 0.16.1 should also be fixed now.

**Comment by gnieuwenhuis** [comment]
scipy 0.16.0 has the same problem - fixed by switching to scipy 0.16.1
